java-vtl-script/src/test/java/no/ssb/vtl/script/functions/VTLFunctionTest.java

```diff
@@ -0,0 +1,30 @@
+package no.ssb.vtl.script.functions;
+
+/*-
+ * ========================LICENSE_START=================================
+ * Java VTL
+ * %%
+ * Copyright (C) 2016 - 2017 Pawel Buczek
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+public interface VTLFunctionTest {
```

Comment:
I'm not sure we need an interface to enforce certain test methods.
If developers end up cramming test of multiple concepts into these methods instead of making new test methods per concept an interface could cause more harm than good IMO